src/ui/molecules/footer/i18n/footer_fr.json

```diff
@@ -0,0 +1,3 @@
+{
+  "brand-link": "Propulsé par"
```

Comment:
I'm not sure about this translation, maybe "Crée par" is more adapted? what do @ccamel et @fredericvilcot think?